assets/css/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: 'Poppins', sans-serif;
+  color: #1a2236;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 4em;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 2%;
+}
+
+.nav-bar > img {
+  width: 6em;
+}
+
+.nav-bar > a > img {
+  width: 1em;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.text-wrapper {
+  padding: 0 5%;
+  margin-bottom: 5%;
+}
```

Comment:
Kindly, you should use flexbox to center align the text.

Reply:
Oh, ok. So this text-wrapper should have a display: flex property and I should justify the content to the center?
Thank you very much for your help.